src/publisher/fragment_logic.py

```diff
@@ -214,6 +214,21 @@ def delete_fragment_update_article(art, key):
         return set_all_article_json(art, quiet=False)
 
 #
+#
+#
+
+def location(av):
```

Comment:
I don't see danger but I do understand where you are coming from. This isn't new code, it's been shifted from the status report that I removed and into fragment logic. There aren't many cases where we don't have the location any more, but dealing with the string 'no-location' in bash is very easy. If something else uses it in future and needs it to be noisy, we can raise exceptions then. Doing a curl on the string 'no-location' produces a wonderfully clear error btw ;)